jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewTest.java

```diff
@@ -109,4 +128,62 @@ public void setDefaultFilterTitleAndDescriptionTest() {
         verify(filterPagedTable, times(3)).saveTabSettings(anyString(), any(HashMap.class));
     }
 
+    @Test
+    public void selectAllButtonAddedTest() {
+        ExtendedPagedTable extendedPagedTable = new ExtendedPagedTable(10,new GridGlobalPreferences("testGrid", new ArrayList<String>(), new ArrayList<String>()));
+
+        view.initBulkActions(extendedPagedTable);
+
+        Iterator it = extendedPagedTable.getRightActionsToolbar().iterator();
+        if(it.hasNext()){
+            Button selectAll = (Button)it.next();
+            assertEquals(selectAll.getType(),IconType.SQUARE_O);
```

Comment:
Just FYI: there is nice alternative to the jUnit asserts - AssertJ, see http://joel-costigliola.github.io/assertj/. It makes it very easy to write readable assertions and also supportss more features than plain jUnit assert.